cfme/services/myservice/__init__.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, appliance, name=None, description=None, vm_name=None):
     @property
     def rest_api_entity(self):
         try:
-            return self.appliance.rest_api.collections.services.get(name=self.name)[0]
```

Comment:
This will no longer raise a `IndexError`, it will raise a `ValueError`

Reply:
good call, thanks! :+1: